client/accept-invite/logged-out-invite/signup-form.jsx

```diff
@@ -11,11 +10,7 @@ import SignupForm from 'components/signup-form'
 import InviteFormHeader from '../invite-form-header'
 import { createAccount, acceptInvite } from '../actions'
 import WpcomLoginForm from 'signup/wpcom-login-form'
-
-/**
- * Module variables
- */
-const debug = debugModule( 'calypso:accept-invite:logged-out' );
```

Comment:
Instead of getting rid of `debug` here, you could actually use it :stuck_out_tongue:. The logic around accepting an invite would be ideal. Not a blocker though.